docs/en/journal.md

```diff
@@ -68,25 +92,31 @@ pageClass: routes
 
 </RouteEn>
 
-### Nature Journal - News
+### News
+
+<RouteEn author="yech1990" example="/nature/news" path="/journal/nature/news" />
+
+### Research Highlight
 
-<RouteEn author="yech1990" example="/nature/nature/news" path="/nature/nature/news" />
+<RouteEn author="yech1990" example="/nature/highlight" path="/journal/nature/highlight" supportScihub="1"/>
 
-### Nature Journal - Research Highlight
+## Proceedings of The National Academy of Sciences (PNAS)
 
-<RouteEn author="yech1990" example="/nature/nature/highlight" path="/nature/nature/highlight" />
+### Latest Articles - Articles by Topic
 
-### Proceedings of The National Academy of Sciences (PNAS) - Latest Articles - ALL
+<RouteEn author="emdoe HenryQW" example="/pnas/Applied Mathematics" path="/journal/pnas/:topic" :paramsDesc="['topic name', 'obtain it from pnas.org (new research in ...). `Latest` will include all.']" supportScihub="1"/>
```

Comment:
Too many preset parameters will increase the requirement for new contributors and the cost of maintenance.

Reply:
On the contrary, keeping 2 copies of the same code snippet decreases the reusability and increases the cost of maintenance.

Reply:
I mean that add `supportScihub` label is not necessary for a simple purpose, which is merely append a prefix to the URL. It should be achieved by passing `:params?` in the router, just in  the same way as other routers. Global parameters and ENV variables should not be invented for special case.

Although a special path `/journal` for the journal section would make the project tidy. But as all the router in this repo are settled by their domain name, those routers for science journal  should not be specialized unless other routes are organized accordingly.

I mean, consistency is more important than convenience in such a huge project.